src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
Similar to the previous comment, consider if setting the margin to 0 is necessary for the `.stars` class, and if so, apply it only to the top or bottom.